packages/data-point/lib/reducer-types/reducer-path/resolve.js

```diff
@@ -1,3 +1,5 @@
+const Promise = require('bluebird')
```

Comment:
Is this a required change?

Reply:
added this because I was getting an error when trying to use `Promise.tap` in the main reducer resolve function - I'm no longer using `tap`, but left this in